CHANGELOG.md

```diff
@@ -85,13 +85,32 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: Biotech Research Area 2 can now be crossed with a Hypermode Wall Boost
 
+##### Impact Crater
+
+##### Magmoor Caverns
+
+- Added: The Sloped R-Jump in Geothermal Core to get to the door to Plasma Processing (commonly referred to as Eagle Jump) is now in logic.
+- Changed: Geothermal Core: Various Puddle Spore requirements now require Before Storage Depot B instead of After.
+
+##### Phazon Mines
+
+- Changed: Central Dynamo: Infinite Speed trick only has one connection and no longer requires Knowledge (Advanced)
```

Comment:
How connections are laid out is irrelevant for logic. So I'd just phrase this as "Getting the item via Infinite Speed now longer requires Knowledge (advanced)"